affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/commitshaproviders/SpecifiedBranchCommit2.kt

```diff
@@ -0,0 +1,17 @@
+package com.dropbox.affectedmoduledetector.commitshaproviders
+
+import com.dropbox.affectedmoduledetector.GitClient
+import com.dropbox.affectedmoduledetector.Sha
+
+class SpecifiedBranchCommit2(private val specifiedBranch: String) : CommitShaProvider {
```

Comment:
Can you please try to think of a better name for this? I understand naming is hard but adding `2` to the end doesn't explain much and makes it a bit confusing

Reply:
Yep, I really agree with u. I thought about it but can't make it better. 

I considered `SpecifiedBranchCommitMergeBase` and `SpecifiedBranchCommit2`
+ explanation in `readme.md` in both cases. 

What do u think about `SpecifiedBranchCommitMergeBase`? Or, maybe can u suggest another name?

Reply:
SpecifiedBranchCommitMergeBase works for me